docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1053,37 +1098,37 @@ Generate a TLS certificate with the Quarkus Dev CA if available.
                       The password of the keystore. Default is 'password'
   -r, --renew         Whether existing certificates will need to be replaced
 ----
-
++
 A `.env` file is also generated when generating the certificate, making the Quarkus dev mode aware of these certificates.
-So, then, if you run your application in dev mode, it will use these certificates:
 
-[source, shell]
+. Run your application in dev mode to use these certificates:
++
+[source,shell]
 ----
 ./mvnw quarkus:dev
 ...
 INFO  [io.quarkus] (Quarkus Main Thread) demo 1.0.0-SNAPSHOT on JVM (powered by Quarkus 999-SNAPSHOT) started in 1.286s. Listening on: http://localhost:8080 and https://localhost:8443
 ----
 
-Now, you can open the Dev UI using HTTPS: `https://localhost:8443/q/dev` or issue a request using `curl`:
-
-[source, shell]
+. Open the Dev UI using HTTPS: `https://localhost:8443/q/dev` or issue a request using `curl`:
++
+[source,shell]
 ----
 curl https://localhost:8443/hello
 Hello from Quarkus REST%
 ----
-
++
 IMPORTANT: A self-signed certificate is generated if the Quarkus Development CA is not installed.
```

Comment:
Any way to get the passive out of this sentence, at least partly?

Reply:
If the Quarkus Development CA is not installed, Quarkus generates a self-signed certificate instead.

Reply:
Great, I will go with this iteration:
Quarkus generates a self-signed certificate if the Quarkus Development CA is not installed.